third_party/utils/hip-config.cmake

```diff
@@ -0,0 +1,252 @@
+
```

Comment:
It looks like we get from other repo. Do we have license issue?

Reply:
it's a file from rocm's official docker, I add license and reference, I'm not sure if it's ok, 
`# Reference: /opt/rocm/hip/lib/cmake/hip/hip-config.cmake from rocm/pytorch:rocm4.0_ubuntu18.04_py3.6`

Reply:
Can we download it directly from AMD repo? We don't need to keep it in our repo.

Reply:
removed this file and try to add a new line in the file of docker directly